src/pages/profitSharing/profile/profile.scss

```diff
@@ -0,0 +1,87 @@
+.traderProfilePage {
```

Comment:
This is the same file for Providers, CT, and PS?

Reply:
yes

Reply:
Ok this will need to be included in the refactor ^^